docs/user/azure/credentials.md

```diff
@@ -0,0 +1,32 @@
+# Service Principal
+
+Before proceeding with the OpenShift install, you should create a service principal with administrative rights for your subscription following the steps
+outlined here:
+
+[Azure: Creating an Service Principal][sp-create]
+
+## Step 1: Create a Service Principal
+
+You can create a Service Principal using the Azure [portal][sp-create-portal] or the Azure [cli][sp-create-cli]
+
+## Step 2: Attach Administrative Role
+
+Azure installer creates new identities for the cluster and therefore requires access to create new roles, and role assignments. Therefore, you will require the service
+principal to have at least `Contributor` and `User Access Administrator` [roles][built-in-roles] assigned in your subscription.
+
+You can create role assignments for your service principal using the Azure [portal][sp-assign-portal] or the Azure [cli][sp-assign-cli]
+
+## Step 3: Acquire Client Secret
+
+You need to save the client secret values to configure your local machine to run the installer. This step is your only opportunity to collect those values.
```

Comment:
additional credentials can be added to the service principal in the azure portal if you didn't capture them.